packages/unity-interface/dcl.ts

```diff
@@ -179,14 +182,51 @@ export async function getPreviewSceneId(): Promise<{ sceneId: string | null; sce
   }
 }
 
-export async function loadPreviewScene(ws?: string) {
-  const { sceneId, sceneBase } = await getPreviewSceneId()
-
-  if (sceneId) {
-    await reloadScene(sceneId)
+export async function loadPreviewScene({ ws, message }: { ws?: string; message?: any }) {
+  if (
+    message &&
+    message.payload &&
+    message.payload.body?.sceneType === 'portable-experience' &&
+    message.payload.body?.sceneId
+  ) {
+    ;(async () => {
+      const sceneId = message.payload.body?.sceneId
+      const url = `${rootURLPreviewMode()}/preview-wearables/${sceneId}`
+      const collection: { data: any[] } = await (await fetch(url)).json()
+
+      if (collection.data.length > 0) {
+        const wearable = collection.data[0]
+        if (!sceneLoading.get(wearable.id)) {
+          await killPortableExperienceScene(wearable.id)
+
+          sceneLoading.set(wearable, true)
+          // This timeout is because the killPortableExperience isn't really async
+          //  and before spawn the portable experience it's neccesary that be kill
+          //  the previous scene
+          // TODO: catch the Scene.unloaded and then call the spawn.
+          await new Promise((resolve) => setTimeout(resolve, 100))
```

Comment:
im pretty sure that we have a sleep fn at some place in this repo.
If now can we create it for future uses ?